src/SprykerEco/Zed/Adyen/Business/Hook/Mapper/MakePayment/KlarnaInvoiceMapper.php

```diff
@@ -17,13 +17,17 @@
 class KlarnaInvoiceMapper extends AbstractMapper
 {
     protected const REQUEST_TYPE = 'klarna';
+    protected const KLARNA_TAX_RATE_MULTIPLIER = 100;
```

Comment:
Looks like smth that we can put into a config

Reply:
Moved to config